src/main/kotlin/org/rust/cargo/project/workspace/CargoWorkspace.kt

```diff
@@ -222,8 +224,13 @@ interface CargoWorkspace {
     }
 
     companion object {
-        fun deserialize(manifestPath: Path, data: CargoWorkspaceData, cfgOptions: CfgOptions): CargoWorkspace =
-            WorkspaceImpl.deserialize(manifestPath, data, cfgOptions)
+        fun deserialize(
+            manifestPath: Path,
+            data: CargoWorkspaceData,
+            cfgOptions: CfgOptions,
+            cargoConfig: CargoConfig,
```

Comment:
Maybe just add default values for `cfgOptions` and `cargoConfig` not to pass them multiple times in tests?